packages/core-utils/src/time.js

```diff
@@ -1,8 +1,9 @@
-import moment from "moment-timezone";
+import { formatDistance, startOfDay, getSeconds, format } from "date-fns";
+import { utcToZonedTime } from "date-fns-tz";
 
 // special constants for making sure the following date format is always sent to
 // OTP regardless of whatever the user has configured as the display format
-export const OTP_API_DATE_FORMAT = "YYYY-MM-DD";
+export const OTP_API_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
This may cause problems elsewhere since this variable is exported.

Reply:
Have created a well-explained second variable